akka-http/src/main/scala/akka/http/javadsl/server/StringUnmarshaller.scala

```diff
@@ -0,0 +1,32 @@
+package akka.http.javadsl.server
+
+import akka.http.scaladsl.unmarshalling.PredefinedFromStringUnmarshallers._
+import akka.http.scaladsl.unmarshalling
+import scala.concurrent.ExecutionContext
+import Unmarshaller.wrap
+import java.util.concurrent.CompletionStage
+
+object StringUnmarshaller {
+  def STRING = sync[String](java.util.function.Function.identity())
+
+  def BYTE: Unmarshaller[String, java.lang.Byte] = wrap(byteFromStringUnmarshaller.map { b ⇒ b })
+  def SHORT: Unmarshaller[String, java.lang.Short] = wrap(shortFromStringUnmarshaller.map { s ⇒ s })
+  def INT: Unmarshaller[String, java.lang.Integer] = wrap(intFromStringUnmarshaller.map { i ⇒ i })
+  def LONG: Unmarshaller[String, java.lang.Long] = wrap(longFromStringUnmarshaller.map { l ⇒ l })
+
+  def BYTE_HEX: Unmarshaller[String, java.lang.Byte] = wrap(HexByte.map { b ⇒ b })
+  def SHORT_HEX: Unmarshaller[String, java.lang.Short] = wrap(HexShort.map { s ⇒ s })
+  def INT_HEX: Unmarshaller[String, java.lang.Integer] = wrap(HexInt.map { i ⇒ i })
+  def LONG_HEX: Unmarshaller[String, java.lang.Long] = wrap(HexLong.map { l ⇒ l })
+
+  def FLOAT: Unmarshaller[String, java.lang.Float] = wrap(floatFromStringUnmarshaller.map { f ⇒ f })
+  def DOUBLE: Unmarshaller[String, java.lang.Double] = wrap(doubleFromStringUnmarshaller.map { d ⇒ d })
+
+  def BOOLEAN: Unmarshaller[String, java.lang.Boolean] = wrap(booleanFromStringUnmarshaller.map { b ⇒ b })
+
+  def UUID = wrap(unmarshalling.Unmarshaller.strict { s: String ⇒ java.util.UUID.fromString(s) })
+
+  def async[B](f: java.util.function.Function[String, CompletionStage[B]]): Unmarshaller[String, B] = Unmarshaller.async(f)
+
+  def sync[B](f: java.util.function.Function[String, B]): Unmarshaller[String, B] = Unmarshaller.sync(f)
+}
```

Comment:
let's not forget to make these into a Java class then (add FIXME until done or tracked somewhere else already)?